var/spack/repos/builtin/packages/ddd/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ddd(AutotoolsPackage, GNUMirrorPackage):
+    """A graphical front-end for command-line debuggers such as GDB, DBX,
+    WDB, Ladebug, JDB, XDB, the Perl debugger, the bash debugger bashdb,
+    the GNU Make debugger remake, or the Python debugger pydb."""
+
+    homepage = "https://www.gnu.org/software/ddd"
+    gnu_mirror_path = "ddd/ddd-3.3.12.tar.gz"
+
+    version('3.3.12', sha256='3ad6cd67d7f4b1d6b2d38537261564a0d26aaed077bf25c51efc1474d0e8b65c')
+
+    depends_on('gdb@4.16:')
+    depends_on('lesstif@0.89:')
+
+    # Needed for OSX 10.9 DP6 build failure:
+    #     https://savannah.gnu.org/patch/?8178
+    patch('https://savannah.gnu.org/patch/download.php?file_id=29114',
+          sha256='aaacae79ce27446ead3483123abef0f8222ebc13fd61627bfadad96016248af6',
+          working_dir='ddd')
+
+    # https://savannah.gnu.org/bugs/?41997
+    patch('https://savannah.gnu.org/patch/download.php?file_id=31132',
+          sha256='f3683f23c4b4ff89ba701660031d4b5ef27594076f6ef68814903ff3141f6714')
+
+    # Patch to fix compilation with Xcode 9
+    #     https://savannah.gnu.org/bugs/?52175
+    patch('https://raw.githubusercontent.com/macports/macports-ports/a71fa9f4/devel/ddd/files/patch-unknown-type-name-a_class.diff',
+          sha256='c187a024825144f186f0cf9cd175f3e972bb84590e62079793d0182cb15ca183',
+          working_dir='ddd')
+
+    def configure_args(self):
+        spec = self.spec
+
+        args = [
+            '--disable-debug',
+            '--disable-dependency-tracking',
+            '--enable-builtin-app-defaults',
+            '--enable-builtin-manual',
+            '--enable-shared' if '+shared' in spec else '--disable-shared',
+            '--enable-static' if '+static' in spec else '--disable-static',
```

Comment:
These variants are missing from the package.

Reply:
Fixing; standby-1